README.md

```diff
@@ -1,43 +1,49 @@
 gobo-awesome-sound
 ==================
 
-An ALSA sound widget for Awesome WM, designed for [http://gobolinux.org](GoboLinux).
+A Pulseaudio sound widget for Awesome WM, designed for [http://gobolinux.org](GoboLinux).
 
 Requirements
 ------------
 
 * Awesome 3.5+
-* `amixer` (from alsa-utils)
-* `alsamixer` and `urxvt` for mixer pop-up
-* `lode-fonts` for a nice-looking mixer pop-up :-)
+* ~~`amixer` (from alsa-utils)~~ `pulseaudio`
+* ~~`alsamixer`~~ ncpamixer (you can use other terminal based pulse mixers) and `urxvt` (will use your default terminal if it aint urxvt) for mixer pop-up
+* `lode-fonts` for a nice-looking mixer pop-up :-) (only used with urxvt)
 
 Using
 -----
 
 Require the module:
 
 
-```
+```lua
 local sound = require("gobo.awesome.sound")
 ```
 
 Create the widget with `sound.new()`:
 
-```
+```lua
 local sound_widget = sound.new()
 ```
 
+You can use a different terminal mixer
```

Comment:
...and terminal? maybe add it here as well.

Reply:
it should read the terminal variable set in rc.lua, don't know what would be a good way to address it